docusaurus/docs/create-a-plugin/develop-a-plugin/manage-npm-dependencies.md

```diff
@@ -0,0 +1,58 @@
+---
+id: manage-npm-dependencies
+title: Manage NPM dependencies
```

Comment:
The title (and its more verbose version) imply this is probably a more generic piece of content than what it covers. 

As I understand it, we're essentially explaining how plugins share runtime dependencies with Grafana? It doesn't seem like there is any action that we're asking the user to take - in which case may make more sense in introduction?

Reply:
I agree it would make more sense to live in introduction. 👍

Reply:
moved, thank you.